neqo-transport/src/recovery.rs

```diff
@@ -590,36 +591,55 @@ impl PtoState {
     }
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub(crate) struct LossRecovery {
     /// When the handshake was confirmed, if it has been.
     confirmed_time: Option<Instant>,
     pto_state: Option<PtoState>,
     rtt_vals: RttVals,
-    cc: CongestionControl,
+    packet_sender: PacketSender,
 
     spaces: LossRecoverySpaces,
 
     qlog: NeqoQlog,
     stats: StatsCell,
 }
 
+impl Default for LossRecovery {
+    fn default() -> Self {
+        Self {
+            confirmed_time: None,
+            pto_state: None,
+            rtt_vals: RttVals::default(),
+            packet_sender: PacketSender::new(CongestionControlAlgorithm::default()),
+            spaces: LossRecoverySpaces::default(),
+            qlog: NeqoQlog::default(),
+            stats: StatsCell::default(),
+        }
+    }
+}
+
 impl LossRecovery {
-    pub fn new(stats: StatsCell) -> Self {
+    pub fn new_with_stats(alg: CongestionControlAlgorithm, stats: StatsCell) -> Self {
```

Comment:
There is no `new` with no stats, so I don't think that you need to rename this one.